python/ray/ml/examples/pytorch/torch_image_example.ipynb

```diff
@@ -0,0 +1,941 @@
+      "91.5%[36m(_execute_read_task pid=14274)\u001b[0m \n",
+      "91.6%[36m(_execute_read_task pid=14274)\u001b[0m \n",
+      "91.6%[36m(_execute_read_task pid=14274)\u001b[0m \n",
+      "93.1%[36m(_execute_read_task pid=14274)\u001b[0m \n",
+      "93.1%[36m(_execute_read_task pid=14274)\u001b[0m \n",
+      "93.1%[36m(_execute_read_task pid=14274)\u001b[0m \n",
+      "93.1%[36m(_execute_read_task pid=14274)\u001b[0m \n",
+      "93.2%[36m(_execute_read_task pid=14274)\u001b[0m \n",
+      "94.5%[36m(_execute_read_task pid=14274)\u001b[0m \n",
+      "94.5%[36m(_execute_read_task pid=14274)\u001b[0m \n",
+      "94.5%[36m(_execute_read_task pid=14274)\u001b[0m \n",
+      "94.6%[36m(_execute_read_task pid=14274)\u001b[0m \n",
+      "94.6%[36m(_execute_read_task pid=14274)\u001b[0m \n",
+      "96.0%[36m(_execute_read_task pid=14274)\u001b[0m \n",
+      "97.6%[36m(_execute_read_task pid=14274)\u001b[0m \n",
+      "99.0%[36m(_execute_read_task pid=14274)\u001b[0m \n",
+      "99.1%[36m(_execute_read_task pid=14274)\u001b[0m \n",
+      "99.1%[36m(_execute_read_task pid=14274)\u001b[0m \n",
+      "99.1%[36m(_execute_read_task pid=14274)\u001b[0m \n",
+      "99.2%[36m(_execute_read_task pid=14274)\u001b[0m \n",
+      "99.2%[36m(_execute_read_task pid=14274)\u001b[0m \n",
+      "99.2%[36m(_execute_read_task pid=14274)\u001b[0m \n",
+      "100.0%36m(_execute_read_task pid=14274)\u001b[0m \n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\u001b[2m\u001b[36m(_execute_read_task pid=14274)\u001b[0m Extracting ./data/cifar-10-python.tar.gz to ./data\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2022-05-09 14:02:49,574\tWARNING read_api.py:248 -- The number of blocks in this dataset (1) limits its parallelism to 1 concurrent tasks. This is much less than the number of available CPU slots in the cluster. Use `.repartition(n)` to increase the number of dataset blocks.\n",
+      "\u001b[2m\u001b[36m(_prepare_read pid=14274)\u001b[0m 2022-05-09 14:02:49,571\tWARNING torch_datasource.py:53 -- `SimpleTorchDatasource` doesn't support parallel reads. The `parallelism` argument will be ignored.\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\u001b[2m\u001b[36m(_execute_read_task pid=14274)\u001b[0m Files already downloaded and verified\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "Dataset(num_blocks=1, num_rows=50000, schema=<class 'tuple'>)"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "\n",
+    "import ray\n",
+    "from ray.data.datasource import SimpleTorchDatasource\n",
+    "import torchvision\n",
+    "import torchvision.transforms as transforms\n",
+    "\n",
+    "transform = transforms.Compose(\n",
```

Comment:
Should these be implemented as a `TorchVisionTransformPreprocessor`?

Reply:
That's not currently a thing, right?